funnel/assets/sass/pages/project.scss

```diff
@@ -459,15 +459,15 @@
       }
     }
     .project-banner__right {
-      width: auto;
+      width: 33%;
     }
   }
   .project-banner.project-banner--inner {
     .project-banner__left--smaller {
       width: 25%;
     }
     .project-banner__right--smaller {
-      width: auto;
+      width: 75%;
```

Comment:
Shouldn't these be `max-width`, with `width` itself remaining `auto`?

Reply:
But in this case we don't want the left column to exceed 67% in any scenario. Setting `width` to `auto` for the right column + `max-width` to 33% would result in a left column larger than 67% when there's no enough content in the right column.

Reply:
The left column's widths are not changing, so it should not be affected by the right column having a max-width. Please test and confirm. This PR should have had before/after screenshots for all scenarios.